_includes/live-demos/premiumskinsandicons-material-classic/index.js

```diff
@@ -2,8 +2,7 @@ tinymce.init({
   selector: 'textarea#premiumskinsandicons-material-classic',
   skin: 'material-classic',
   content_css: [
```

Comment:
Another one that could be simplified to remove the array.

Reply:
Fixed in https://github.com/tinymce/tinymce-docs/pull/1645/commits/8bc36cb9e57a4c890e01fa0a70ebc3484a22a7bb